Dockerfile.fedora-26

```diff
@@ -0,0 +1,78 @@
+
+RUN echo 'd0a98590c74bfe36af0ce006f7b25fa60246aecb /etc/tmpfiles.d/opendnssec.conf' | sha1sum --quiet -c && mv -v /etc/tmpfiles.d/opendnssec.conf /usr/lib/tmpfiles.d/opendnssec.conf
+RUN echo 'd2090704dc077e75a70420a5ee2534b7148bb35d /etc/systemd/system/sssd.service.d/journal.conf' | sha1sum --quiet -c && rm -vf /etc/systemd/system/sssd.service.d/journal.conf
+RUN find /etc/systemd/system/* '!' -name '*.wants' | xargs rm -rvf
+RUN for i in basic.target network.service netconsole.service ; do rm -f /usr/lib/systemd/system/$i && ln -s /dev/null /usr/lib/systemd/system/$i ; done
+RUN rm -fv /usr/lib/systemd/system/sysinit.target.wants/*
+RUN echo 'disable *' > /usr/lib/systemd/system-preset/10-container-disable.preset
+
+RUN echo LANG=C > /etc/locale.conf
+
+RUN /sbin/ldconfig -X
+
+COPY init-data /usr/local/sbin/init
+COPY ipa-server-configure-first ipa-server-status-check exit-with-status ipa-volume-upgrade-* /usr/sbin/
+COPY install.sh uninstall.sh /bin/
+RUN mv /bin/hostnamectl /bin/hostnamectl.orig
+RUN mv /usr/bin/domainname /usr/bin/domainname.orig
+ADD hostnamectl-wrapper /bin/hostnamectl
+ADD hostnamectl-wrapper /usr/bin/domainname
+RUN chmod -v +x /usr/local/sbin/init /usr/sbin/ipa-server-configure-first /usr/sbin/ipa-server-status-check /usr/sbin/exit-with-status /usr/sbin/ipa-volume-upgrade-* /bin/install.sh /bin/uninstall.sh /bin/hostnamectl /usr/bin/domainname
+COPY container-ipa.target ipa-server-configure-first.service ipa-server-upgrade.service ipa-server-update-self-ip-address.service /usr/lib/systemd/system/
+RUN rmdir -v /etc/systemd/system/multi-user.target.wants \
+	&& mkdir /etc/systemd/system/container-ipa.target.wants \
+	&& ln -s /etc/systemd/system/container-ipa.target.wants /etc/systemd/system/multi-user.target.wants
+
+RUN systemctl set-default container-ipa.target
+RUN systemctl enable ipa-server-configure-first.service
+
+COPY exit-via-chroot.conf /usr/lib/systemd/system/systemd-poweroff.service.d/
+
+RUN groupadd -g 288 kdcproxy ; useradd -u 288 -g 288 -c 'IPA KDC Proxy User' -d '/var/lib/kdcproxy' -s '/sbin/nologin' kdcproxy
+
+COPY atomic-install-help /usr/share/ipa/
+COPY volume-data-list volume-data-mv-list volume-data-autoupdate /etc/
+RUN set -e ; cd / ; mkdir /data-template ; cat /etc/volume-data-list | while read i ; do echo $i ; if [ -e $i ] ; then tar cf - .$i | ( cd /data-template && tar xf - ) ; else mkdir -p /data-template$( dirname $i ) ; fi ; mkdir -p $( dirname $i ) ; if [ "$i" == /var/log/ ] ; then mv /var/log /var/log-removed ; else rm -rf $i ; fi ; ln -sf /data${i%/} ${i%/} ; done
+RUN rm -rf /var/log-removed
+RUN sed -i 's!^d /var/log.*!L /var/log - - - - /data/var/log!' /usr/lib/tmpfiles.d/var.conf
+# Workaround 1286602
+RUN mv /usr/lib/tmpfiles.d/journal-nocow.conf /usr/lib/tmpfiles.d/journal-nocow.conf.disabled
+RUN mv /data-template/etc/dirsrv/schema /usr/share/dirsrv/schema && ln -s /usr/share/dirsrv/schema /data-template/etc/dirsrv/schema
```

Comment:
This workaround for dirsrv should not be necessary, in fact it breaks the subsequent deployment of the image:

```console
2017-05-24T16:09:28Z DEBUG Process finished, return code=1
2017-05-24T16:09:28Z DEBUG stdout=[17/05/24:16:09:28] - [Setup] Info Could not copy file '/etc/dirsrv/schema/schema' to '/etc/dirsrv/slapd-EXAMPLE-TEST/schema/schema'.  Error: Is a directory
Could not copy file '/etc/dirsrv/schema/schema' to '/etc/dirsrv/slapd-EXAMPLE-TEST/schema/schema'.  Error: Is a directory
```